src/app/styles/_hilite.scss

```diff
@@ -2,7 +2,7 @@
  * String highligting via :hilite 
  */
 
-.exist {
+a[exist] {
```

Comment:
"The" and "a" seems to get classified with the `.monster` and then `.exists` is nested within, which ends up with weird highlighting:

![CleanShot 2020-08-16 at 14 44 33@2x](https://user-images.githubusercontent.com/69156/90344489-30501380-dfcf-11ea-8cd5-bd15b41110cb.png)

We could fix that at the CSS level though easily, just wanna make sure that's intended.

Reply:
Yeah, this is something that is inherent to the game feed itself, so I think I should stick to that as closely as possible and only fix "errors" from the feed